cpp/src/slice2cs/Gen.cpp

```diff
@@ -772,15 +772,28 @@ Slice::CsVisitor::writeDocComment(const ContainedPtr& p, const string& generated
         writeDocLines(_out, "summary", comment->overview());
     }
 
+    StringList remarks;
+    if (comment)
+    {
+        remarks = comment->remarks();
+    }
     if (!generatedType.empty())
     {
-        _out << nl << "/// <remarks>" << "The Slice compiler generated this " << generatedType << " from Slice "
-             << p->kindOf() << " <c>" << p->scoped() << "</c>.";
+        if (!remarks.empty())
+        {
+            remarks.emplace_back("");
```

Comment:
I don't think blank lines are expected/correct in C# doc-comments. We should look into using `<para>`.

Reply:
I removed the blank line and replace it with a `para` tag.
See https://github.com/zeroc-ice/ice/pull/3962#issuecomment-2881104196